challenge-programs/high-performance-tidb-challenge.md

```diff
@@ -1,32 +1,65 @@
 
-*   Team up with other participants
-*   Report sre-bot bugs
-*   Ask any questions concerning this challenge
+* Team up with other participants
+* Report sre-bot bugs
+* Ask any questions concerning this challenge
+
+## Participation Notes
+
+Glad you are interested! Before you start hacking, these are something you should know to get you started quickly.
+
+### About team-up
+
+* You need to team up before submitting the [application form](https://forms.pingcap.com/f/high-performance-challenge-en). One team could only select one workload for optimization.
+* Up to five participants are allowed in a team; cross team-up is not allowed.
+* If you have trouble in team-up, give a shout in the Slack channel for help. Or you can sign up as a one-person team.
+
+### About challenge points
+
+* Your team only earns points when you resolve the issues that you created for your target workload optimization. If the issues resolved are not relevant, there will be no points granted.
+* The team points are the sum of the points of all the tasks completed by your team members.
+* The leaderboard in real time does not indicate the final team rankings. See [Evaluation criteria](#evaluation-criteria) for all the factors at play.
+* The organizing committee determines the challenge points for the created issues under each workload. The points are determined according to the team's estimated work.
+
+### About mentors
+
+* Only one mentor is allowed in a team and one mentor could only provide instructions to one team.
+* The team that needs mentoring could contact the organizing committee in the Slack channel.
+
+### Evaluation criteria
+
+The results of the challenge will be evaluated in teams. The organizing committee will pull the latest working branches of each team at 18:00 on Nov.15 (China Standard Time) to evaluate workload optimization results and calculate the total PR points of each team. Besides, each team has 7 minutes to present their teamwork for the optimization on Nov.21.
+
+**The final ranking is determined by:**
+
+* Performance improvement (65%): performance improvement (percentage) of the working branch compared with the base Master branch
+* PR points of the team (30%): the validated PRs merged into Master
+* Presentation (5%): presentation performance
 
 ## Before participation
 
-*   See [Join GitHub](https://github.com/join) to create a GitHub account
-*   [Install Git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git) in your local environment
-*   Check [Set up Git](https://help.github.com/en/github/getting-started-with-github/set-up-git) to configure your Git to access GitHub
+1   See [Join GitHub](https://github.com/join) to create a GitHub account
+2  [Install Git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git) in your local environment
+3   Check [Set up Git](https://help.github.com/en/github/getting-started-with-github/set-up-git) to configure your Git to access GitHub
```

Comment:
The format is broken here. It needs to be `1.` etc.

Otherwise LGTM, thanks for sharing & good luck in the program!

Reply:
Thanks.